gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java

```diff
@@ -62,7 +62,12 @@
             "-XX:HeapDumpPath=var/log",
             // Set DNS cache TTL to 20s to account for systems such as RDS and other
             // AWS-managed systems that modify DNS records on failover.
-            "-Dsun.net.inetaddr.ttl=20");
+            "-Dsun.net.inetaddr.ttl=20",
+            // Disable RFC 5077 TLS session ticket support due to incompatibility with common implementations.
```

Comment:
rather than common implementations, can we explicitly call out the problem of each server node just making up it's own session key (and then nginx breaking), with a specific mentionof PDS-123520?

Reply:
I don't think the servers using different keys is the problem we're hitting with nginx, that would be the case if we had dns resolution across several nodes or transparent load balancing. However nginx knows each upstream is a unique entity with its own key. We can name PDS-123520 and nginx though